Kitodo-API/src/main/java/org/kitodo/api/validation/filestructure/FileStructureValidationInterface.java

```diff
@@ -11,23 +11,48 @@
 
 package org.kitodo.api.validation.filestructure;
 
+import java.io.IOException;
 import java.net.URI;
+import java.util.Collection;
 
 import org.kitodo.api.validation.ValidationInterface;
 import org.kitodo.api.validation.ValidationResult;
+import org.xml.sax.SAXException;
 
-/** Validates a given xml file against a given xsd schema. */
+/** Validates a given XML file against a given xsd schema. */
 public interface FileStructureValidationInterface extends ValidationInterface {
 
     /**
-     * validates a xml file at a given uri against a xsd at a given location.
-     * 
+     * Validates an XML file at a given uri against a xsd at a given location.
+     *
      * @param xmlFileUri
      *            The file to validate.
      * @param xsdFileUri
      *            The location of the schema to validate against.
      * @return A ValidationResult, with result boolean and resultMessages.
      */
-    ValidationResult validate(URI xmlFileUri, URI xsdFileUri);
+    ValidationResult validate(URI xmlFileUri, URI xsdFileUri) throws SAXException, IOException;
+
+    /**
+     * Validates a String containing XML against a xsd at a given location.
+     *
+     * @param xmlString
+     *            String containing XML content to validate.
+     * @param xsdFiles
+     *            Collection of URIs pointing to the schema files to validate against.
+     * @return A ValidationResult, with result boolean and resultMessages.
+     */
+    ValidationResult validate(String xmlString, Collection<URI> xsdFiles) throws IOException, SAXException;
+
+    /**
+     * Validates an XML file at a given uri against a xsd at a given location.
+     *
+     * @param xmlString
+     *            The string with XML content to validate.
+     * @param xsdFileUri
+     *            The location of the schema to validate against.
+     * @return A ValidationResult, with result boolean and resultMessages.
+     */
+    ValidationResult validate(String xmlString, URI xsdFileUri) throws SAXException, IOException;
```

Comment:
Incomplete JavaDoc for thrown exceptions.

Reply:
done